WeatherApp/WeatherApp/Core/AppDelegate.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  AppDelegate.swift
+//  WeatherApp
+//
+//  Created by Phan Quyen on 06/08/2025.
+//
+
+import UIKit
+import CoreData
+
+@main
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        print("App did finish launching")
```

Comment:
remove toàn bộ print khi đẩy PR em nhé, nếu muốn keep lại thì phải dùng debugPrint